libvast/src/system/configuration.cpp

```diff
@@ -261,10 +261,7 @@ caf::error merge_environment(record& config) {
                                            "variable {}={} to VAST value: {}",
                                            key, value, x.error()));
         } else {
-          return caf::make_error(ec::parse_error,
-                                 fmt::format("could not parse environment "
-                                             "variable {} value '{}': {}",
-                                             key, value, config_value.error()));
+          config[*config_key] = std::string{value};
```

Comment:
I'm fine with the fix, but it needs a bug fix changelog entry.

Reply:
made one. If you're fine with that one or have a wording proposal please take over the merge.